### DISCUSSION TITLE
new plugin: file-icons

### DISCUSSION THREAD

**Discussion by rejetto** [rejetto/hfs#280] [Show and tell]
freely decide icons
https://github.com/rejetto/file-icons

**Comment by GeorgeGeorgeGe** [comment]
it works perfect.
can you make it accept assigning the same icon file to more than one extension at the same time?
for example xlsx,xls,xlsm,...>>>  the same icon file.
Thanks

**Comment by rejetto** [comment]
it is already so, but you have to separate with | instead of ,

**Comment by PH5HP** [comment]
Hi,

I assigned to most file extensions a new icon now, but there are some with the default icon. They have sometimes no extension or they have an non standard extension.

Is there a way to assign a new icon to the files which does not have a new file icon? To replace the default doscument icon? PSE take a look at my screenshot.

Kind regards,
Henk - ftp.ph5hp.nl
 
![image](https://github.com/rejetto/hfs/assets/140949730/757cccba-3500-4c2b-9e94-317c41e5e4f4)

**Comment by rejetto** [comment]
I will add such option

**Reply by PH5HP** [reply]
Thank you :-)

**Comment by rejetto** [comment]
version 2.1 available @PH5HP

**Reply by PH5HP** [reply]
Hi,

Thank you... It works nicely...

Kind regards,
Henk - ftp.ph5hp.nl
